x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/messages/Messages.java

```diff
@@ -74,8 +74,7 @@
     public static final String DATA_FRAME_ANALYTICS_AUDIT_STARTED_LOADING_DATA = "Started loading data";
     public static final String DATA_FRAME_ANALYTICS_AUDIT_STARTED_ANALYZING = "Started analyzing";
     public static final String DATA_FRAME_ANALYTICS_AUDIT_STARTED_WRITING_RESULTS = "Started writing results";
-    public static final String DATA_FRAME_ANALYTICS_CANNOT_UPDATE_IN_CURRENT_STATE =
-        "Cannot update analytics [{0}] while its status is {1}";
+    public static final String DATA_FRAME_ANALYTICS_CANNOT_UPDATE_IN_CURRENT_STATE = "Cannot update analytics [{0}] unless it''s stopped";
```

Comment:
type: remove double quote

Reply:
I believe it is needed so that a single quote renders properly. Please look at the other messages, like:
```
    public static final String JOB_AUDIT_REVERTED = "Job model snapshot reverted to ''{0}''";
```

Reply:
or remove contraction and change to `it is` :D